modules/jcl/pom.xml

```diff
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
-            <version>1.1.1</version>
+            <version>1.2</version>
```

Comment:
I don't understand this. Why do we still have this dependency if the ticket states that it should be removed?

Reply:
The goal is to remove commons-logging from ignite-spring. ignite-jcl still requires this dependency (e.g. what if user wants to run GG without ignite-spring but with ignite-jcl). Task to remove ignite-jcl is already in backlog for 8.9.x

Reply:
ok, got it